Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -937,3 +938,36 @@ However, all new configuration by the operator will be blocked with this cleanup
 Rook waits for the deletion of PVs provisioned using the cephCluster before proceeding to delete the
 cephCluster. To force deletion of the cephCluster without waiting for the PVs to be deleted, you can
 set the `allowUninstallWithVolumes` to true under `spec.CleanupPolicy`.
+
+## Ceph Config
+
+!!! warning
```

Comment:
1. If there are too many hints and warnings, then they become easy to overlook IMO. I'd suggest the only warning or hint we need in this section should be that these settings are to be used at one's own risk since Rook does not validate if they are going to cause some issue with the cluster. Everything else can just be a normal paragraph. 
2. Documenting the unset for removed settings can just be a paragraph at the end of the section. It seems like a corner case, but not a warning.